### PR TITLE
feat: color code app types

### PR DESCRIPTION
**Pull request by aherzog-auth0**
### Description

Color codes app types when displayed in list and key-value pairs. 

Screen Shot 2021-03-26 at 2 37 09 PM

### References
https://github.com/auth0/auth0-cli/issues/106

### CODE REVIEW

File: internal/display/apps.go [hunk 1]
```diff
@@ -305,3 +305,22 @@ func interfaceSliceToString(s []interface{}) []string {
 	}
 	return res
 }
+
+func applyColor(a string) string {
```
Comment (jfatta) [inline internal/display/apps.go:309]: do you think we can unify the _friendly_ names on a single place as constants?
It is also hardcoded here: https://github.com/auth0/auth0-cli/blob/71463818fa582c1ebdca035c2d4d922b1d958da9/internal/display/apps.go#L266-L279

**Review by jfatta — APPROVED** [review]
Looks good 👍 
I don't have strong opinions about the colors anyway 🕶️ 

Just a comment about constants, other than that feel free to merge.

File: internal/display/apps.go (hunk 1 above)
Comment (aherzog-auth0) [reply internal/display/apps.go:309]: yea good call 👍

**Comment by Widcket** [comment]
Thanks!

**Pull request merged**